Makefile

```diff
@@ -127,6 +135,12 @@ start: build-backend
 	bash -c "trap 'fuser -k 9002/tcp' EXIT; \
 					./plugin-backend -port 9002 & cd web && npm run start" 
 
+.PHONY: start-standalone
```

Comment:
May be start-standalone should also depends on install-frontend otherwise, dependency might be missing.